### PR TITLE
13-transport.Rmd: minor fixes

### PR DESCRIPTION
**Pull request by florisvdh**

### CODE REVIEW

File: 13-transport.Rmd [hunk 1]
```diff
@@ -51,7 +51,7 @@ Routes (which can be represented as single linestrings or multiple short *segmen
 They can be represented as geographic features (typically short segments of road that add up to create a full network) or structured as an interconnected graph, with the level of traffic on different segments referred to as 'flow' by transport modelers [@hollander_transport_2016]
 
 Another key level is **agents**, mobile entities like you and me and vehicles that enable us to move such as bikes and buses.
-These can be represented computationally in software such as [MATSim](http://www.matsim.org/) and [A/B Street](https://github.com/a-b-street/abstreet), which represent the dynamics of transport systems using an agent-based modeling (ABM)\index{agent-based modeling} frameworks, usually at high levels of spatial and temporal resolution [@horni_multi-agent_2016]. 
+These can be represented computationally in software such as [MATSim](http://www.matsim.org/) and [A/B Street](https://github.com/a-b-street/abstreet), which represent the dynamics of transport systems using an agent-based modeling (ABM)\index{agent-based modeling} framework, usually at high levels of spatial and temporal resolution [@horni_multi-agent_2016].
```
Comment (Robinlovelace) [inline 13-transport.Rmd:54]: :+1:

File: 13-transport.Rmd [hunk 2]
```diff
@@ -93,7 +93,7 @@ tm_shape(region_all[1, ], bbox = region_all) +
   tm_basemap(server = leaflet::providers$Esri.WorldTopoMap)
 ```
 
-```{r bristol, echo=FALSE, fig.cap="Bristol's transport network represented by colored lines for active (green), public (railways, black) and private motor (red) modes of travel. Black border lines represent the inner city boundary (highlighted in yellow) and the larger Travel To Work Area (TTWA).", fig.scap="Bristol's transport network."}
+```{r bristol, echo=FALSE, fig.cap="Bristol's transport network represented by colored lines for active (green), public (railways, blue) and private motor (red) modes of travel. Black border lines represent the inner city boundary (highlighted in yellow) and the larger Travel To Work Area (TTWA).", fig.scap="Bristol's transport network."}
```
Comment (Robinlovelace) [inline 13-transport.Rmd:96]: Aha yes that was a recent-ish update.

**Review by Robinlovelace — APPROVED** [review]
Thanks @florisvdh :pray:

**Comment by florisvdh** [comment]
Always a pleasure to learn from your great book! I should read it cover-to-cover one day.

**Pull request merged**